### DISCUSSION TITLE
How to get mime type image/png returned in the base64 output

### DISCUSSION THREAD

**Discussion by thammer67** [chillerlan/php-qrcode#272] [General]
I am trying to use php-qrcode to generate a base64 encoded PNG that I can pass to FPDF's Image() function as I need to include a QR code in a large PDF I am generating.

I have tried to follow the QRGdImage example (among others) to generate the string but no matter what I try, it always comes back with 'data:image/svg+xml' and FPDF complains saying "FPDF error: Not a PNG file". 

I am using PHP 8.2.5 and php-qrcode 5.0.2 on Debian 12 (Bookworm)

How can I get php-qrcode to return "data:image/png" so I can accomplish this?

Thanks!

**Comment by codemasher** [comment]
Hi, it appears you're using examples from the main/dev branch, instead of the ones for your installed version, see the bold text over here: https://github.com/chillerlan/php-qrcode?tab=readme-ov-file#documentation

**Comment by thammer67** [comment]
Sorry. I missed that bit. I mistakenly assumed the latest release branch would contain the lastest code commits. 

I removed php-qrcode and installed dev-main#0ca3d2377f8d01b788c738d46f9bf72d9b17083d (which is the latest commit in main) and it's now working fine. Thanks!
